vendor/github.com/vmware/photon-controller-go-sdk/SSPI/sspi.go

```diff
@@ -0,0 +1,276 @@
+// +build windows
+
+package SSPI
+
+import (
+	"fmt"
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	secur32_dll           = syscall.NewLazyDLL("secur32.dll")
+	initSecurityInterface = secur32_dll.NewProc("InitSecurityInterfaceW")
```

Comment:
is this dll and proc guaranteed to exist on all supported windows platforms? (since you're initializing it in the `init()` function, a panic would prevent the binary from starting, even if photon was never used